modules/prefixer.js

```diff
@@ -242,15 +244,44 @@ var getPrefixedPropertyName = function (property) {
   return prefixedPropertyCache[property] = workingProperty;
 };
 
+// We are un-prefixing values before checking for isUnitlessNumber,
+// otherwise we are at risk of being in a situation where someone
+// explicitly passes something like `MozBoxFlex: 1` and that will
+// in turn get transformed into `MozBoxFlex: 1px`.
+var _getUnprefixedProperty = function (property) {
+  var noPrefixProperty = property;
+
+  vendorPrefixes.forEach(prefix => {
```

Comment:
Ah yes, I didn't even think of that - I was too caught up about the fact that you can't use a `break` with `forEach`! Fix incoming :)